ix-dev/community/esphome/item.yaml

```diff
@@ -0,0 +1,7 @@
+categories:
+- productivity
```

Comment:
Is that the right category?

Reply:
i copied this from the home assistant app. Is there a list of categories somewhere?

Reply:
I don't know. This just stood out. I asked in https://github.com/truenas/apps/discussions/1409.

Reply:
I found https://github.com/truenas/apps/blob/master/ix-dev/community/zigbee2mqtt/item.yaml#L2C3-L2C18

```suggestion
- home-automation
```